partiql-types/src/main/kotlin/org/partiql/value/PartiQLValue.kt

```diff
@@ -471,10 +471,13 @@ public abstract class StructValue<T : PartiQLValue> : PartiQLValue, Sequence<Pai
 
     public abstract val fields: Sequence<Pair<String, T>>?
```

Comment:
I kind of think this should just be a list or array. Array makes more sense honestly because of fixed size rather than an immutable List, but either way not a sequence

Reply:
I believe you're tackling this in your PR for modifying PartiQL Value.